Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -540,7 +540,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
             }
 
             var relativeFile = CommonUtils.GetRelativeFilePath(this.FullPathToChildren, fileNode.Url);
-            if (this._analyzer.Project != null && this._analyzer.Project.Limits.IsPathExceedNestingLimit(relativeFile)) {
+            if (this._analyzer != null && this._analyzer.Project != null && this._analyzer.Project.Limits.IsPathExceedNestingLimit(relativeFile)) {

```

Comment:
We've actually already taken this change, so I presume this is why we're seeing merge conflicts. Things should work a little better if you remove this file from the commit. 
https://github.com/Microsoft/nodejstools/blob/master/Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs#L581-L584